packages/jest-runtime/src/index.js

```diff
@@ -267,40 +273,68 @@ class Runtime {
     return cliOptions;
   }
 
+  teardown() {
```

Comment:
Can we call this `reset` and call it in the constructor also for less duplication?